hawkular-inventory-parent/hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -18,19 +18,19 @@
 
 import java.util.EnumMap;
 
-import org.hawkular.inventory.api.model.AbstractElement;
-import org.hawkular.inventory.api.model.Blueprint;
-import org.hawkular.inventory.api.model.DataEntity;
-import org.hawkular.inventory.api.model.Environment;
-import org.hawkular.inventory.api.model.Feed;
-import org.hawkular.inventory.api.model.MetadataPack;
-import org.hawkular.inventory.api.model.Metric;
-import org.hawkular.inventory.api.model.MetricType;
-import org.hawkular.inventory.api.model.OperationType;
-import org.hawkular.inventory.api.model.Relationship;
-import org.hawkular.inventory.api.model.Resource;
-import org.hawkular.inventory.api.model.ResourceType;
-import org.hawkular.inventory.api.model.Tenant;
+import org.hawkular.inventory.api.oldmodel.AbstractElement;
```

Comment:
(Forgive me if I make some basic questions, most of them are related I'm not familiar with the details of the inventory project)
Question:
Do we need to main the oldmodel ?

Reply:
I can remove it, it's too distracting in code review...